base_converter/__init__.py

```diff
@@ -3,26 +3,39 @@
 """Convert representations of numbers.
 
 Synopsis:
-    <trigger> <src base> <dest base> <src>
-    <bin|oct|dec|hex> <number> [padding]"""
+    <trigger> <dest base> <src>
+    <number> [padding]
+
+    where <src> is a literal of the form '0bXXX' (binary), '0XXX' (octal), 
+    or '0xXXX' (hexadecimal)."""
 
 import numpy as np
+from collections import defaultdict
 
-from albertv0 import *
+from albertv0 import Item, ClipAction
 
 __iid__ = "PythonInterface/v0.1"
 __prettyname__ = "Base Converter"
-__version__ = "1.1"
+__version__ = "2.0"
```

Comment:
This is a bit messy. actually all the versions should be 0.* but well as they are they are 1.* . please use it. within each version they should be backward compatible.

Reply:
Sure, I can change it to 1.2. My rationale for 2.0 was that the syntax change is not backwards compatible. Should I change it so both the old style and the one I'm suggesting work?

Reply:
Thats fine, but the major version is intented to match the interface of the core. Users need no backward compatibiity for the human-machine interface :D

Reply:
Sounds reasonable enough! I've reverted it to 1.1.